### PR TITLE
i18n(fr) Update `middleware.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)
Update translation for middleware.mdx file

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by thomasbnt — APPROVED** [review]
LGTM! 🚀

File: src/content/docs/fr/guides/middleware.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ Le middleware est disponible dans les projets SSG et SSR Astro.
 
 1. Créez `src/middleware.js|ts` (Alternativement, vous pouvez créer `src/middleware/index.js|ts`.)
 
-2. Dans ce fichier, exportez une fonction [`onRequest()`](#onrequest). Il ne doit pas s'agir d'une exportation par défaut.
+2. Dans ce fichier, exportez une fonction [`onRequest()`]/fr/reference/api-reference/#onrequest)2. Dans ce fichier, exportez une fonction [`onRequest()`](/fr/reference/api-reference/#onrequest) à laquelle on peut passer un objet [`context`](#objet-context) et une fonction `next()`. Il ne doit pas s'agir d'une exportation par défaut.
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/middleware.mdx:18]:
```suggestion
2. Dans ce fichier, exportez une fonction [`onRequest()`](/fr/reference/api-reference/#onrequest) à laquelle on peut passer un objet [`context`](#objet-context) et une fonction `next()`. Il ne doit pas s'agir d'une exportation par défaut.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**